docs/FAQ.md

```diff
@@ -56,13 +56,13 @@ No
 
 ### What is the policy on supporting end-of-life(EOL) Ruby versions?
 
-We will target the latest the EOL Ruby, currently that is 1.9.3.
+We will target the latest the EOL Ruby, please check the link below to see current EOL version.
```

Comment:
Change to...

We will target the latest EOL Ruby version, please check the link below the see the current EOL version.